src/gui/SetupDialog.cpp

```diff
@@ -233,6 +235,8 @@ SetupDialog::SetupDialog(ConfigTabs tab_to_open) :
 		m_oneInstrumentTrackWindow, SLOT(toggleOneInstrumentTrackWindow(bool)), true);
 	addLedCheckBox("Show sidebar on the right-hand side", gui_tw, counter,
 		m_sideBarOnRight, SLOT(toggleSideBarOnRight(bool)), true);
+	addLedCheckBox("Use solo legacy behavior", gui_tw, counter,
```

Comment:
Maybe `Solo mutes Automation Tracks` or something along these lines? Someone coming into LMMS in the future would have to go look up what legacy behavior is (or test it out which they may not have automation tracks in and be confused). Changing it to explain what it will do I think is better than just saying `legacy behavior`.

Reply:
You're right, I'll change it to something more descriptive!

Reply:
I used `Mute automation tracks during solo` as a description of the setting now!